nservicebus/upgrades/rabbitmq-3to4.md

```diff
@@ -26,17 +26,22 @@ See [Tuning](/nservicebus/operations/tuning.md).
 ### PrefetchCount
 
 
-`PrefetchCount` is obsolete and `EndpointConfiguration.LimitMessageProcessingConcurrencyTo` should be used instead. See [Tuning](/nservicebus/operations/tuning.md).
+[PrefetchCount](/nservicebus/rabbitmq/configuration-api.md#configuring-rabbitmq-transport-to-be-used-prefetchcount) is obsolete and `EndpointConfiguration.LimitMessageProcessingConcurrencyTo` should be used instead. See [Tuning](/nservicebus/operations/tuning.md).
 
 
 ### DequeueTimeout
 
-`DequeueTimeout` is deprecated as the RabbitMQ Transport message pump does not require a timeout.
+[DequeueTimeout](/nservicebus/rabbitmq/configuration-api.md#configuring-rabbitmq-transport-to-be-used-dequeuetimeout) is deprecated as the RabbitMQ Transport message pump does not require a timeout.
```

Comment:
What needs to be done for users that were using the `DequeueTimeout`? Can it be safely removed? If so can we specify that?